components/Snackbar/src/MDCSnackbarAlignment.h

```diff
@@ -0,0 +1,30 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import <Foundation/Foundation.h>
+
+/**
+ States used to configure snackbar alignment.
+ */
+typedef NS_ENUM(NSInteger, MDCSnackbarAlignment) {
```

Comment:
I was using `NSTextAlignment` as a guide. `NSTextAlignment` aligns _text_ in a container, like a `UILabel`, for example. Similarly, I figured `MDCSnackbarAlignment` could align a snackbar in a container. I'm open to `MDCSnackbarScreenAlignment` though, definitely.

Reply:
Ah, interesting. I just realized that when I look at MDC APIs I tend to subconsciously drop both the MDC and the component name, so I read this API as simply “alignment” and wasn’t sure what was being aligned.

NSTextAlignment on the other hand I read as “text alignment”.

This may just be me though, so I don’t feel super strongly about adding “screen” to the API.

Reply:
It wasn't brought up in any of the previous comments--how do we feel about introducing such a big API addition (a whole new header file!) for something that was originally labeled "good first issue"? It seemed a little bold to me, I just couldn't think of a better way haha 🙈